yellowbrick/features/jointplot.py

```diff
@@ -129,13 +129,14 @@ def __init__(self, ax=None, feature=None, target=None,
                  xy_plot='hist', xy_args=None,
                  size=6, ratio=5, space=.2, **kwargs):
 
-        #check matplotlib version - needs to be version 2.0.0
-        if mpl.__version__ == "2.0.0":
+        # Check matplotlib version - needs to be version 2.0.0 or greater.
+        mpl_vers_maj = int(mpl.__version__.split(".")[0])
+        if mpl_vers_maj >= 2:
```

Comment:
This is really minor, but I would propose a refactor as follows:

```python
if mpl_vers_maj < 2:
    warnings.warn((
        "{} requires matplotlib major version 2 or greater. "
        "Please upgrade."
     ).format(self.__class__.__name__))
```

I'm just a bit worried that in the current else form the logic may be accidentally broken by another contributor.